packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -218,20 +218,21 @@ service SpaceInvitationsService {
   rpc CancelInvitation(CancelInvitationRequest) returns (google.protobuf.Empty);
 }
 
-enum NetworkMode {
+enum ConnectionState {
   OFFLINE = 0;
   ONLINE = 1;
 }
 
-message GetNetworkModeResponse {
-  NetworkMode mode = 1;
+// TODO(mykola): Broden API.
+message NetworkStatus {
+  ConnectionState state = 1;
 }
 
-message SetNetworkModeRequest {
-  NetworkMode mode = 1;
+message SetNetworkOptionsRequest {
+  ConnectionState state = 1;
 }
 
 service NetworkService {
-  rpc getNetworkMode(google.protobuf.Empty) returns (GetNetworkModeResponse);
-  rpc SetNetworkMode(SetNetworkModeRequest) returns (google.protobuf.Empty);
+  rpc SubscribeToNetworkStatus(google.protobuf.Empty) returns (stream NetworkStatus);
```

Comment:
check other protos that return streams -- I think we use Get -- vs. the API where we may say createSubscription. consistency over everything.

Reply:
There is no consistency, we use `Get<something>`, `Subscribe<something>`, `SubscribeTo<something>`, and `Query<something>` (in echo), and sometimes naming does not correspond with the return type.